keanu-python/examples/coal_mining.py

```diff
@@ -12,15 +12,15 @@ def __init__(self):
     def model(self):
         start_year, end_year = (self._data.index.min(), self._data.index.max())
 
-        with kn.Model() as m:
-            m.switchpoint = kn.UniformInt(int(start_year), int(end_year + 1))
+        with Model() as m:
+            m.switchpoint = UniformInt(int(start_year), int(end_year + 1))
 
-            m.early_rate = kn.Exponential(1.0)
-            m.late_rate = kn.Exponential(1.0)
+            m.early_rate = Exponential(1.0)
+            m.late_rate = Exponential(1.0)
 
             m.years = np.array(self._data.index)
-            m.rates = kn.DoubleIf([1, 1], m.switchpoint > m.years, m.early_rate, m.late_rate)
-            m.disasters = kn.Poisson(m.rates)
+            m.rates = DoubleIf([1, 1], m.switchpoint > m.years, m.early_rate, m.late_rate)
```

Comment:
Nit: can we make the [1, 1] in to a tuple? ie (1, 1).  Lists should ideally only be used for things you want to be mutable.

Reply:
Good call.